requirements/base.txt

```diff
@@ -64,8 +68,9 @@ click-repl==0.2.0
     # via celery
 cmislib-maykin==0.7.4
     # via drc-cmis
-commonground-api-common==1.13.3
+commonground-api-common[testutils]==2.1.0
```

Comment:
I think we should release a new version of `commonground-api-common` first, before merging this PR, so that you can pin the latest version here